Lab1_Operationalizing_Pytorch_with_Mlflow/src/bert-classifier/requirements.txt

```diff
@@ -1,11 +1,10 @@
-boto3
-cloudpickle
-google-cloud-storage
+torch>=1.6.0
 mlflow>=1.12.0
-mlflow-torchserve
+torchserve
+torch-model-archiver
 numpy
-pandas
+transformers
 sklearn
-torchtext>=0.7.0
 tqdm
-transformers==3.5.0
\ No newline at end of file
+torchtext>=0.7.0
+pandas
```

Comment:
Might not be clear from this PR, but this is just the original `requirements.txt`